### ISSUE TITLE
Importer returns standard ubl for CreditNote: Why not EStandard ubl_creditnote?

### ISSUE THREAD

**Issue by melo0187** [ZUGFeRD/mustangproject#632]
Since UBL has entirely different types for Invoice and CreditNote it would facilitate further processing (e.g. with https://github.com/phax/ph-ubl) a lot if `ZUGFeRDInvoiceImporter`'s method for returning the standard would return `ubl` for UBL Invoices and `ubl_creditnote` for UBL CreditNote.

The enum `EStandard` already contains the entry `ubl_creditnote`, which is why I was expecting to get this as return from  `getStandard()` for CreditNote documents and not `ubl`.

https://github.com/ZUGFeRD/mustangproject/blob/9b846ff4b68281a5e01ef181ca9cb471d84a74f2/library/src/main/java/org/mustangproject/ZUGFeRD/ZUGFeRDInvoiceImporter.java#L991-L992

**Comment by jstaerk** [comment]
i guess I just overlooked that instance, do the tests still run when corrected?

**Comment by melo0187** [comment]
> i guess I just overlooked that instance, do the tests still run when corrected?
TL;DR: I believe they do.

In an attempt to provide a quick answer to the question I pulled the latest master to run the test.
First this required fixing compile errors in `ZF2EdgeTest`, because it claims to override `getPaymentMeansCode` and `getPaymentMeansInformation`, which do not exist.
Then I did actually see a test failure in `testEEISI_300_cii_Import(org.mustangproject.ZUGFeRD.ZF2ImporterTest)`.
However, this test fails already on master as is.
Returning `EStandard.ubl_creditnote` for `rootNode.equals("CreditNote")` does not add any test failures.
